internal/image/install/install.go

```diff
@@ -18,6 +18,6 @@ limitations under the License.
 package install
 
 type Installation struct {
-	Bootloader    string `yaml:"bootloader"`
-	KernelCmdLine string `yaml:"kernelCmdLine"`
+	Bootloader    string `json:"bootloader"`
```

Comment:
guess this is because sigs yaml package takes json annotations... so, the change of the package wasn't completely smooth, isn't it? 😆

Reply:
Well, I saw that we already used the `json:` annotations elsewhere so I opted to use the same style in all of the code :sweat_smile: